### PR TITLE
Novaya Rossiyskaya Imperiya(NRI) ERT

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

This was a request from our Russian downstreams.

Adds a new ERT to the game, the NRI, a specialised military force for a wide range of applications. 
image

There are 3 unit types:

The commander:
image

The commander is in charge of the platoon and has a cap to show that.

The soldier:
image

The soldier is a common force, answering to the Commander.

The heavy:
image

A heavy variant of the soldier equipped with heavy armour.

The standard equipment for all ERT members:
image

They all come with pan slavic.

## Changelog

:cl: Cobalt, Ramirez, Flavrius
add: A new ERT type, the NRI.
/:cl:

### CODE REVIEW

**Comment by gonenoculer5** [comment]
Probably should make sure they can speak pan-slavic if they're gilgamesh.

**Comment by Gandalf2k15** [comment]
> Probably should make sure they can speak pan-slavic if they're gilgamesh.

done

File: modular_skyrat/modules/gilgamesh_colonial_confederation/code/head.dm
Comment (Draganfrukts) [inline modular_skyrat/modules/gilgamesh_colonial_confederation/code/head.dm]:
```suggestion
	name = "commander's beret"
```

File: modular_skyrat/modules/gilgamesh_colonial_confederation/code/head.dm
Comment (Draganfrukts) [inline modular_skyrat/modules/gilgamesh_colonial_confederation/code/head.dm]:
```suggestion
	desc = "A heavy Russian combat helmet with a strong ballistic visor. Alt+click to adjust."
```

File: modular_skyrat/modules/gilgamesh_colonial_confederation/code/suit.dm
Comment (Draganfrukts) [inline modular_skyrat/modules/gilgamesh_colonial_confederation/code/suit.dm]:
```suggestion
	desc = "Modern body armor designed to protect the torso from bullets, shrapnel and blunt force. A lead-plated layer offers a very limited amount of radiation protection."
```

File: modular_skyrat/modules/gilgamesh_colonial_confederation/code/suit.dm
Comment (Draganfrukts) [inline modular_skyrat/modules/gilgamesh_colonial_confederation/code/suit.dm]:
```suggestion
	name = "REDUT-H armour system"
```

File: modular_skyrat/modules/gilgamesh_colonial_confederation/code/suit.dm
Comment (Draganfrukts) [inline modular_skyrat/modules/gilgamesh_colonial_confederation/code/suit.dm]:
```suggestion
	desc = "A strong set of full-body armor designed for harsh environments. It has nothing in it to aid the user's movement."
```

**Review by Draganfrukts — CHANGES_REQUESTED** [review]
Pending the GUBMAN® Seal of Approval™

**Comment by Iamgoofball** [comment]
Do we really need more ERTs?

File: modular_skyrat/modules/gilgamesh_colonial_confederation/code/head.dm
Comment (Iamgoofball) [inline modular_skyrat/modules/gilgamesh_colonial_confederation/code/head.dm]: Hahahahaha, no. Absolutely not.

File: modular_skyrat/modules/gilgamesh_colonial_confederation/code/head.dm
Comment (Iamgoofball) [inline modular_skyrat/modules/gilgamesh_colonial_confederation/code/head.dm]: Species flag, not like this.

File: modular_skyrat/modules/gilgamesh_colonial_confederation/code/suit.dm
Comment (Iamgoofball) [inline modular_skyrat/modules/gilgamesh_colonial_confederation/code/suit.dm]: Hahahahaha, no. Absolutely not.

File: modular_skyrat/modules/gilgamesh_colonial_confederation/code/suit.dm
Comment (Iamgoofball) [inline modular_skyrat/modules/gilgamesh_colonial_confederation/code/suit.dm]: Drop this to sec armor values.

File: modular_skyrat/modules/gilgamesh_colonial_confederation/code/suit.dm
Comment (Iamgoofball) [inline modular_skyrat/modules/gilgamesh_colonial_confederation/code/suit.dm]: species flag

File: modular_skyrat/modules/gilgamesh_colonial_confederation/code/uniform.dm
Comment (Iamgoofball) [inline modular_skyrat/modules/gilgamesh_colonial_confederation/code/uniform.dm]: No. Remove the armor from this.

**Comment by hugmeorelse** [comment]
We don't have GCC.

**Comment by Stalkeros2** [comment]
> We don't have GCC.

That sucks ngl, better add them in asap.

**Comment by Lukaster42** [comment]
> > We don't have GCC.
> 
> That sucks ngl, better add them in asap.

Indeed.

File: modular_skyrat/modules/novaya_ert/code/ert.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/novaya_ert/code/ert.dm]: I'd suggest renaming the typepath to reflect the name change, otherwise gcc won't mean anything to anyone.

File: modular_skyrat/modules/novaya_ert/code/head.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/novaya_ert/code/head.dm]: Again, I recommend changing gcc to something else.

File: modular_skyrat/modules/novaya_ert/code/head.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/novaya_ert/code/head.dm:41]: Does this work even when Tesharis would spawn with it?

File: modular_skyrat/modules/novaya_ert/code/id.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/novaya_ert/code/id.dm]: GCC is still mentioned here too.

File: modular_skyrat/modules/novaya_ert/code/outfit.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/novaya_ert/code/outfit.dm]: More gcc

File: modular_skyrat/modules/novaya_ert/code/suit.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/novaya_ert/code/suit.dm]: NRI - The GCC Strikes Back

File: modular_skyrat/modules/novaya_ert/code/survival_pack.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/novaya_ert/code/survival_pack.dm]: Guess who's there again?

File: modular_skyrat/modules/novaya_ert/code/uniform.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/novaya_ert/code/uniform.dm]: Guess what

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Got a few name changes left to go.

File: modular_skyrat/modules/novaya_ert/code/head.dm
Comment (Gandalf2k15) [reply modular_skyrat/modules/novaya_ert/code/head.dm:41]: Should do yes

File: modular_skyrat/modules/novaya_ert/code/head.dm
Comment (GoldenAlpharex) [inline modular_skyrat/modules/novaya_ert/code/head.dm]: Would be nice to have a more significative name and some documentation on what it does.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Just one tiny final nitpick.

**Review by GoldenAlpharex — APPROVED** [review]
Good to merge once CI passes.

**Comment by REBOLUTION228-a11** [comment]
Aren't you the guy who connected to the White Dream server, that had the same russian ERT-analogue on it before your PR with NRI?

**Pull request merged**
